aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -201,6 +229,7 @@ function healthcheck {
 
 	if [[ "$EXISTING_RULE" == "200" &&
 		  "$EXISTING_PROFILE" == "200" &&
+		  "$EXISTING_QUALYS" == "200" &&
```

Comment:
This won't work if the user isn't using Qualys (which we are making optional, determined by whether they set the URL or not).

Please copy the [example from ARTIFACTORY_URL](https://github.com/anshupande/dcos-scripts/blob/b4d79cfb6d32b62f8f349c1e6a68498c20430889/aqua-ethos-setup/aqua_ethos.sh#L224)